compiler/ballerina-lang/build.gradle

```diff
@@ -41,12 +41,14 @@ description = 'Ballerina - Lang'
 checkstyleMain {
     exclude 'io/ballerina/projects/internal/bala/BalaJson.java'
     exclude 'io/ballerina/projects/internal/bala/PackageJson.java'
+    exclude 'io/ballerina/projects/internal/bala/CompilerPluginJson.java'
```

Comment:
Why do we need to exclude this ?

Reply:
Need to exclude it due to these variable names:
```java
private String plugin_class;
    private List<String> dependency_paths;